articles/getting_started/using_development_nuget_packages.md

```diff
@@ -0,0 +1,131 @@
+# Using the Development Nuget Packages
```

Comment:
Header needs the front matter used by the doc generator and remove the h1 element (it should be the title)